TodoApp.IntegrationTesting/System/Controller/UserControllerTest.cs

```diff
@@ -0,0 +1,142 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Moq;
+using TodoApp.Controllers;
+using Microsoft.Extensions.Options;
+using TodoApp.Models;
+using FluentAssertions;
+using TodoApp.IntegrationTesting.Fixtures;
+
+namespace TodoApp.IntegrationTesting.System.Controller
+{
+    public class UserControllerTest
+    {
+        [Fact]
+        public async Task Get_OnSuccess_ReturnStatusCode200()
+        {
+            // Arrange
+            var sut = new UserController();
+
+            // Act
+            var result= (OkResult)await sut.GetAync();
+
+            // Assert
+            result.StatusCode.Should().Be(200);
```

Comment:
It's better to test the actual result in addition to status code, where ever you return a result.